assets/javascripts/discourse/components/ai-llms-list-editor.gjs

```diff
@@ -14,11 +16,60 @@ import AiLlmEditor from "./ai-llm-editor";
 
 export default class AiLlmsListEditor extends Component {
   @service adminPluginNavManager;
+  @service router;
+
+  sanitizedTranslationKey(id) {
+    return id.replace(/\./g, "-");
+  }
 
   get hasLLMElements() {
     return this.args.llms.length !== 0;
   }
 
+  get preConfiguredLlms() {
+    let options = [
+      {
+        id: "none",
+        name: I18n.t(`discourse_ai.llms.preconfigured.fake`),
+        provider: "fake",
+      },
+    ];
+
+    const llmsContent = this.args.llms.content.map((llm) => ({
+      provider: llm.provider,
+      name: llm.name,
+    }));
+
+    this.args.llms.resultSetMeta.presets.forEach((llm) => {
+      if (llm.models) {
+        llm.models.forEach((model) => {
+          const id = `${llm.id}-${model.name}`;
+          const isConfigured = llmsContent.some(
+            (content) =>
+              content.provider === llm.provider && content.name === model.name
+          );
+
+          if (!isConfigured) {
+            options.push({
+              id,
+              name: model.display_name,
+              provider: llm.provider,
+            });
+          }
+        });
+      }
+    });
+
+    return options;
+  }
+
+  @action
+  transitionToLlmEditor(llm) {
```

Comment:
I would just name it llmTemplate instead of llm, so I dont have to specify it after.

Also it's more clear, if it's a template, let's say it's a template.